init.lua

```diff
@@ -328,26 +349,38 @@ local function commandHandler(...)
     local args = {...}
     if #args == 1 then
         if args[1] == 'sell' and not loot.Terminate then
-            doSell = true
+            if not mq.TLO.Cursor() then
```

Comment:
this seems out of place. since you moved it to not take the item name in the set item as sell, now this is overloaded where if you do lootutils sell with something on cursor, its setting that item as sell, but if cursor is empty then it will run the sell stuff to vendor routine.
maybe the do sell stuff needs a new name so they don't conflict, idk.

Reply:
i contemplated having setsell for setting items to sell as an argument.
sell just seems much more intuitive. 
could use sellstuff for actuall selling.

Reply:
yea sellstuffs fine. or vendor maybe. whatever works, i prefer changing the command and keeping sell as the loot option.